conf/airframes/tudelft/disco_modified.xml

```diff
@@ -0,0 +1,635 @@
+    <define name="MIN_CIRCLE_RADIUS" value="25." unit="m"/> <!-- TODO: determine optimum minimum -->
+
+    <define name="CARROT" value="4." unit="s"/>
+
+    <!--UNLOCKED_HOME_MODE if set to TRUE means that HOME mode does not get stuck.
+    If not set before when you would enter home mode you had to flip a bit via the GCS to get out. -->
+    <define name="UNLOCKED_HOME_MODE" value="TRUE"/>
+    <!-- RC_LOST_MODE means that if your RC Transmitter signal is not received anymore in the autopilot, e.g. you switch it off
+    or  fly a long range mission you define the wanted mode behaviour here.
+    If you do not define it, it defaults to flying to the flightplan HOME -->
+    <define name="RC_LOST_MODE" value="AP_MODE_AUTO2"/>
+
+    <!-- TODO: SET some modem values with multipoint also the $AC_ID -->
+    <!--  Here XBEE init will be misused to set SiK Si10xx based modems as the Hope and RFdesign -->
+    <!--  <define name="XBEE_INIT" value="ATS17=$AC_ID\rATS16=134\rAT&W\rATZ\r" type="string"/>  -->
+    <!--  <define name="NO_XBEE_API_INIT" value="TRUE"/> -->
+ </section>
+
+<!-- ************************ GLS_APPROACH ************************* -->
+ <!-- see: -->
+  <section name="GLS_APPROACH" prefix="APP_">
+    <define name="DISTANCE_AF_SD" value="20" unit="m"/>
+    <define name="ANGLE" value="4" unit="deg"/> <!-- TODO: determine optimal value -->
+    <define name="INTERCEPT_AF_SD" value="80" unit="m"/> <!--Watch it after landing airspeed = 0 thus aircraft throttles up again, not good -->
+    <!--<define name="INTERCEPT_RATE" value="0.624" unit="m/s/s"/>-->
+    <define name="TARGET_SPEED" value="12.0" unit="m/s"/> <!-- Via airspeed -->
+  </section>
+
+<!-- ********************** GCS SPECIFICS ************************** -->
+  <section name="GCS">
+    <define name="AC_ICON" value="flyingwing"/>
+    <define name="ALT_SHIFT_PLUS_PLUS" value="40"/>
+    <define name="ALT_SHIFT_PLUS" value="20"/>
+    <define name="ALT_SHIFT_MINUS" value="-20"/>
+  </section>
+
+  <!-- ************************ SIMU ******************************** -->
+  <section name="SIMU">
+    <define name="JSBSIM_MODEL" value="&quot;Malolo1&quot;"/>
+    <!--define name="JSBSIM_INIT"   value="&quot;Malolo1-IC&quot;"/-->
+    <define name="JSBSIM_LAUNCHSPEED" value="11.0"/>
+    <define name="WEIGHT" value="1."/>
+    <define name="JSBSIM_IR_ROLL_NEUTRAL" value="RadOfDeg(0.)"/>
+    <define name="JSBSIM_IR_PITCH_NEUTRAL" value="RadOfDeg(0.)"/>
+    <define name="YAW_RESPONSE_FACTOR" value=".9"/>   <!--default 1.-->
+    <define name="PITCH_RESPONSE_FACTOR" value="1."/> <!--default 1.-->
+    <define name="ROLL_RESPONSE_FACTOR" value="20."/> <!--default 15-->
+  </section>
+
+<!-- ************************ JSBSIM ******************************* -->
+  <section name="SIMULATOR" prefix="NPS_">
+    <define name="JSBSIM_MODEL" value="&quot;Malolo1&quot;" type="string"/>
```

Comment:
you shouldn't mix `&quot;` and `type="string"` (better keep the `type` attribute.